src/bcf/record.rs

```diff
@@ -157,6 +157,12 @@ impl Record {
     ///
     /// When empty, returns `b".".to_vec()`.
     pub fn id(&self) -> Vec<u8> {
+        // TODO: do we have to use `&mut self` here because of the unpack?!
```

Comment:
Yes, I think so. Otherwise, we could get a race condition. Would it make sense to put the unpack into the the constructor? Or does it cause a big overhead?